zjit/src/backend/lir.rs

```diff
@@ -1477,6 +1477,16 @@ impl Assembler
                 }
             }
 
+            if let Some(idx) = vreg_idx {
```

Comment:
It's also nice to add a comment above this to explain this is for VReg never used by other instructions.